--- base/queues.py.orig	2013-06-21 08:57:55.000000000 +0200
+++ base/queues.py	2013-07-09 16:05:04.000000000 +0200
@@ -87,6 +87,9 @@ def check_user_groups():
 
 # This function adds the groups ('lp') to user
 def add_group(core, mode, passwordObj):
+# Deactivated via "return True" because it causes security issues
+# see https://bugs.launchpad.net/bugs/1197416
+    return True
     result = False
     add_user_to_group = core.get_distro_ver_data('add_user_to_group', '')
     if add_user_to_group:
