diff -up hplip-3.14.6/prnt/hpcups/LJZjStream.cpp.logdir hplip-3.14.6/prnt/hpcups/LJZjStream.cpp
--- hplip-3.14.6/prnt/hpcups/LJZjStream.cpp.logdir	2013-05-10 12:03:36.000000000 +0200
+++ hplip-3.14.6/prnt/hpcups/LJZjStream.cpp	2013-05-14 14:57:19.460942143 +0200
@@ -42,10 +42,11 @@
 #include "LJZjStream.h"
 #include "Utils.h"
 #include "hpjbig_wrapper.h"
 #include "utils.h"
+#include <limits.h>
 
 #define ZJC_BAND_HEIGHT    100

 LJZjStream::LJZjStream () : Encapsulator ()
 {
     memset(&m_PM, 0, sizeof(m_PM));
                          
--- hplip-3.14.6/prnt/hpcups/SystemServices.cpp.logdir	2014-06-05 12:40:11.614447986 +0200
+++ hplip-3.14.6/prnt/hpcups/SystemServices.cpp	2014-06-05 12:42:44.460023500 +0200
@@ -31,6 +31,7 @@
 #include "CommonDefinitions.h"
 #include "SystemServices.h"
 #include "utils.h"
+#include <limits.h>
 
 SystemServices::SystemServices(int iLogLevel, int job_id, char* user_name) : m_iLogLevel(iLogLevel)
 {
   