diff -up hplip-3.20.11/base/password.py.omv~ hplip-3.20.11/base/password.py
--- hplip-3.20.11/base/password.py.omv~	2021-02-03 14:56:24.319391449 +0100
+++ hplip-3.20.11/base/password.py	2021-02-03 14:56:50.772689515 +0100
@@ -57,6 +57,7 @@ AUTH_TYPES = {'mepis': 'su',
               'boss': 'su',
               'lfs': 'su',
               'manjarolinux': 'sudo',
+              'openmandrivalx': 'sudo',
               }
 
 
