diff -Naur hplip-3.13.8/data/rules/56-hpmud.rules hplip-3.13.8.tpg/data/rules/56-hpmud.rules
--- hplip-3.13.8/data/rules/56-hpmud.rules	2013-08-07 06:02:33.000000000 +0000
+++ hplip-3.13.8.tpg/data/rules/56-hpmud.rules	2013-08-19 11:24:05.938453838 +0000
@@ -1,6 +1,6 @@
 # HPLIP udev rules file. Notify console user if plugin support is required for this device.
 
-ACTION!="add", GOTO="hpmud_rules_end"
+ACTION="remove", GOTO="hpmud_rules_end"
 SUBSYSTEM=="ppdev", OWNER="root", GROUP="lp", MODE="0660"
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", GOTO="hpmud_usb_rules"
 
diff -Naur hplip-3.13.8/data/rules/56-hpmud_sysfs.rules hplip-3.13.8.tpg/data/rules/56-hpmud_sysfs.rules
--- hplip-3.13.8/data/rules/56-hpmud_sysfs.rules	2013-08-07 06:02:33.000000000 +0000
+++ hplip-3.13.8.tpg/data/rules/56-hpmud_sysfs.rules	2013-08-19 11:23:51.843080742 +0000
@@ -1,6 +1,6 @@
 # HPLIP udev rules file. Notify console user if plugin support is required for this device.
 
-ACTION!="add", GOTO="hpmud_rules_end"
+ACTION="remove", GOTO="hpmud_rules_end"
 SUBSYSTEM=="ppdev", OWNER="root", GROUP="lp", MODE="0660"
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", GOTO="hpmud_usb_rules"
 
