diff -up hplip-3.23.3/base/password.py.80~ hplip-3.23.3/base/password.py
--- hplip-3.23.3/base/password.py.80~	2023-04-29 13:11:22.410101546 +0200
+++ hplip-3.23.3/base/password.py	2023-04-29 13:13:25.652152141 +0200
@@ -279,9 +279,10 @@ class Password(object):
         return AuthType, AuthCmd
 
     def __get_password_utils_ui(self):
-        (os_name,_ver) = utils.get_distro_name()
-        os_name = os_name.lower().replace(" ","")
-        distro_name = get_distro_std_name(os_name)
+        import distro
+        os_name = distro.id()
+        _ver = distro.version()
+        distro_name = os_name
         if self.__authType == "sudo":
             AuthType, AuthCmd = 'sudo', 'sudo %s'
         else:
