diff -uraN hplip-3.23.12/prnt/hpps/hppsfilter.c omv-hplip-3.23.12/prnt/hpps/hppsfilter.c
--- hplip-3.23.12/prnt/hpps/hppsfilter.c	2023-11-27 16:06:40.000000000 +0100
+++ omv-hplip-3.23.12/prnt/hpps/hppsfilter.c	2023-12-04 15:32:38.950053749 +0100
@@ -108,7 +108,7 @@
     if (fd < 0) {
         temp_filename[0] = '\0';
         fprintf(stderr, "ERROR: Unable to open temp file %s\n", temp_filename);
-        return 1;
+        return;
     }
 
     ptempbooklet_file = fdopen(fd, mode);
diff -uraN hplip-3.23.12/scan/sane/hpaio.c omv-hplip-3.23.12/scan/sane/hpaio.c
--- hplip-3.23.12/scan/sane/hpaio.c	2023-11-27 16:06:41.000000000 +0100
+++ omv-hplip-3.23.12/scan/sane/hpaio.c	2023-12-04 15:35:01.210335716 +0100
@@ -414,20 +414,34 @@
 
 extern void sane_hpaio_close(SANE_Handle handle)
 {
-    if (strcmp(*((char **)handle), "MARVELL") == 0)
-       return marvell_close(handle);
-    if (strcmp(*((char **)handle), "SOAP") == 0)
-       return soap_close(handle);
-    if (strcmp(*((char **)handle), "SOAPHT") == 0)
-       return soapht_close(handle);
-    if (strcmp(*((char **)handle), "LEDM") == 0)
-       return ledm_close(handle);
-    if (strcmp(*((char **)handle), "SCL-PML") == 0)
-       return sclpml_close(handle);
-    if (strcmp(*((char **)handle), "ESCL") == 0)
-       return escl_close(handle);
-    if (strcmp(*((char **)handle), "ORBLITE") == 0)
-       return orblite_close(handle);
+    if (strcmp(*((char **)handle), "MARVELL") == 0) {
+       marvell_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SOAP") == 0) {
+       soap_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SOAPHT") == 0) {
+       soapht_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "LEDM") == 0) {
+       ledm_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SCL-PML") == 0) {
+       sclpml_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "ESCL") == 0) {
+       escl_close(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "ORBLITE") == 0) {
+       orblite_close(handle);
+       return;
+    }
 }  /* sane_hpaio_close() */
 
 extern const SANE_Option_Descriptor * sane_hpaio_get_option_descriptor(SANE_Handle handle, SANE_Int option)
@@ -547,20 +561,34 @@
 /* Note, sane_cancel is called normally not just during IO abort situations. */
 extern void sane_hpaio_cancel( SANE_Handle handle )
 {
-    if (strcmp(*((char **)handle), "MARVELL") == 0)
-       return marvell_cancel(handle);
-    if (strcmp(*((char **)handle), "SOAP") == 0)
-       return soap_cancel(handle);
-    if (strcmp(*((char **)handle), "SOAPHT") == 0)
-       return soapht_cancel(handle);
-    if (strcmp(*((char **)handle), "LEDM") == 0)
-       return ledm_cancel(handle);
-    if (strcmp(*((char **)handle), "SCL-PML") == 0)
-       return sclpml_cancel(handle);
-    if (strcmp(*((char **)handle), "ESCL") == 0)
-       return escl_cancel(handle);
-    if (strcmp(*((char **)handle), "ORBLITE") == 0)
-       return orblite_cancel(handle);
+    if (strcmp(*((char **)handle), "MARVELL") == 0) {
+       marvell_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SOAP") == 0) {
+       soap_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SOAPHT") == 0) {
+       soapht_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "LEDM") == 0) {
+       ledm_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "SCL-PML") == 0) {
+       sclpml_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "ESCL") == 0) {
+       escl_cancel(handle);
+       return;
+    }
+    if (strcmp(*((char **)handle), "ORBLITE") == 0) {
+       orblite_cancel(handle);
+       return;
+    }
 }  /* sane_hpaio_cancel() */
 
 extern SANE_Status sane_hpaio_set_io_mode(SANE_Handle handle, SANE_Bool nonBlocking)
