--- hplip-3.14.6/prnt/hpijs/hpijs.cpp.hpijs-marker-supply	2014-06-03 08:31:00.000000000 +0200
+++ hplip-3.14.6/prnt/hpijs/hpijs.cpp	2014-06-05 11:42:16.472644343 +0200
@@ -589,6 +589,7 @@ int main (int argc, char *argv[], char *
    int status = EXIT_FAILURE;
    int ret, n, i, kn=0, width, k_width;
    char user_name[32]={0,};
+   int low_marker = 0;
         
    openlog("hpijs", LOG_PID,  LOG_DAEMON);
 
