diff -uraN hplip-3.21.10/base/password.py omv-hplip-3.21.10/base/password.py
--- hplip-3.21.10/base/password.py	2021-10-28 23:31:45.000000000 +0200
+++ omv-hplip-3.21.10/base/password.py	2021-11-11 02:20:26.109798070 +0100
@@ -36,6 +36,7 @@
               'debian': 'su',
               'suse': 'su',
               'mandriva': 'su',
+              'openmandrivalx': 'sudo',
               'fedora': 'su',
               'fedora28': 'sudo',
               'redhat': 'su',
