diff --git a/installer/pluginhandler.py b/installer/pluginhandler.py
index 1af8d85..5c498ab 100755
--- a/installer/pluginhandler.py
+++ b/installer/pluginhandler.py
@@ -462,9 +462,21 @@ class PluginHandle(object):
                     log.error("Target file %s does not exist. File copy failed." % trg)
                     continue
                 else:
-                    # Make plugin .so files readable/executable by all users
-                    os.chmod(trg, 0o755)
-                    log.debug("Set plugin file to 0o755: %s" % trg)
+                    if trg.endswith('.rules'):
+                        os.chmod(trg, 0o644)
+                        with open(trg, 'r') as udev_f:
+                            content = udev_f.read()
+
+                        content = content.replace('libsane_rules_end', 'libsane_usb_rules_end')
+                        content = content.replace('GROUP="scanner", ', '')
+
+                        with open(trg, 'w') as udev_f:
+                            udev_f.write(content)
+
+                    else:
+                        # Make plugin .so files readable/executable by all users
+                        os.chmod(trg, 0o755)
+                        log.debug("Set plugin file to 0o755: %s" % trg)
 
                 if link:
                     if os.path.exists(link):
