--- hplip-3.15.4/data/rules/56-hpmud.rules	2015-06-01 19:52:50.228259942 -0700
+++ hplip-3.15.4-udev-on-action/data/rules/56-hpmud.rules	2015-06-01 19:42:43.196444435 -0700
@@ -1,6 +1,6 @@
 # HPLIP udev rules file. Notify console user if plugin support is required for this device.
 
-ACTION!="add", GOTO="hpmud_rules_end"
+ACTION!="remove", GOTO="hpmud_rules_end"
 SUBSYSTEM=="ppdev", OWNER="root", GROUP="lp", MODE="0664"
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", GOTO="hpmud_usb_rules"
 
