diff -up hplip-3.18.12/Makefile.am.004~ hplip-3.18.12/Makefile.am
--- hplip-3.18.12/Makefile.am.004~	2018-12-19 00:37:04.778596619 +0100
+++ hplip-3.18.12/Makefile.am	2018-12-19 00:40:21.019455407 +0100
@@ -47,7 +47,7 @@ if !HPLIP_CLASS_DRIVER
 hpdir = $(cupsbackenddir)
 hp_PROGRAMS = hp
 hp_SOURCES = prnt/backend/hp.c
-hp_LDADD = libhpmud.la $(DBUS_LIBS)
+hp_LDADD = libhpmud.la -lcups $(DBUS_LIBS)
 hp_LDFLAGS = $(libhpmud_la_LDFLAGS)
 hp_CFLAGS = $(DBUS_CFLAGS)
 endif #HPLIP_CLASS_DRIVER 
