diff -Naur hplip-3.14.6.orig/prnt/hpcups/HPCupsFilter.cpp hplip-3.14.6/prnt/hpcups/HPCupsFilter.cpp
--- hplip-3.14.6.orig/prnt/hpcups/HPCupsFilter.cpp	2014-07-10 18:00:10.623877105 +0400
+++ hplip-3.14.6/prnt/hpcups/HPCupsFilter.cpp	2014-07-10 18:00:28.512876654 +0400
@@ -597,7 +597,7 @@
     char hpPreProcessedRasterFile[MAX_FILE_PATH_LEN]; //temp file needed to store raster data with swaped pages.
 
 
-    sprintf(hpPreProcessedRasterFile, "%s/hp_%s_cups_SwapedPagesXXXXXX",CUPS_TMP_DIR, m_JA.user_name);
+    snprintf(hpPreProcessedRasterFile, sizeof(hpPreProcessedRasterFile), "%s/hp_%s_cups_SwapedPagesXXXXXX",CUPS_TMP_DIR, m_JA.user_name);
 
 
     while (cupsRasterReadHeader2(cups_raster, &cups_header))
diff -Naur hplip-3.14.6.orig/prnt/hpcups/SystemServices.cpp hplip-3.14.6/prnt/hpcups/SystemServices.cpp
--- hplip-3.14.6.orig/prnt/hpcups/SystemServices.cpp	2014-07-10 18:00:10.645877105 +0400
+++ hplip-3.14.6/prnt/hpcups/SystemServices.cpp	2014-07-10 18:03:01.156872806 +0400
@@ -38,8 +38,8 @@
     if (iLogLevel & SAVE_OUT_FILE)
     {
         char    fname[MAX_FILE_PATH_LEN];
-        sprintf(fname, "%s/hpcups_%s_out_job%d_XXXXXX",CUPS_TMP_DIR, user_name, job_id);
-        
+        snprintf(fname, sizeof(fname), "%s/hpcups_%s_out_job%d_XXXXXX",CUPS_TMP_DIR, user_name, job_id);
+
         createTempFile(fname, &m_fp);
         if (m_fp)
         {
